### ISSUE TITLE
crash when loading config file.

### ISSUE THREAD

**Issue by cmal** [paritytech/parity#8248]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>  parity --chain myGenesis.json --config parity.config.toml
> - **Which Parity version?**: Parity/v1.9.5-stable-ff821da-20180321/x86_64-linux-gnu/rustc1.24.1
> - **Which operating system?**:  ubuntu 16.04
> - **How installed?**: sudo dpkg -i parity_1.9.5_ubuntu_amd64.deb
> - **Are you fully synchronized?**: no
> - **Which network are you connected to?**: private
> - **Did you try to restart the node?**: no

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

```
Loading config file from parity.config.toml

====================

stack backtrace:
   0:     0x557b24a8491c - <no info>

Thread 'main' panicked at 'Logger is initialized only once; qed: "Cannot write to log file given: /var/log/parity.log"', /checkout/src/libcore/result.rs:916
```

**Comment by 5chdn** [comment]
Please share your config file.

**Comment by Tbaut** [comment]
solved in [gitter](https://gitter.im/paritytech/parity), @cmal didn't have the rights to access this file. This was solved by using
```
[misc]
log_file = "/home/Your_user/eth/my-chain/parity.log"
```
